08-Placing-Barriers-in-the-Simulation-Box/04-Wall-potential-MD.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<p>Add the word "mobile": </p><p>".... place <strong>mobile</strong> particles in the center..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>